alertmanager/Dockerfile

```diff
@@ -14,10 +14,33 @@
 # limitations under the License.
 
 # https://hub.docker.com/r/prom/alertmanager/tags?page=1&ordering=last_updated
-FROM prom/alertmanager:v0.27.0
-LABEL maintainer="Astronomer <humans@astronomer.io>"
+FROM prom/alertmanager:v0.27.0 AS UPSTREAM
 
+FROM quay.io/astronomer/ap-base:3.18.6-2
```

Comment:
@pgvishnuram Why did we change this to a multi-stage build?

Reply:
im unable to find a way to add update-ca-certificates in busybox base image

Reply:
That makes sense. This looks like the Dockerfile: https://github.com/prometheus/alertmanager/blob/main/Dockerfile I think using a two-stage build is fine in this case.